docs/metrics/customizing-the-sdk/README.md

```diff
@@ -367,11 +367,31 @@ MyFruitCounter.Add(1, new("name", "apple"), new("color", "red"));
 AnotherFruitCounter.Add(1, new("name", "apple"), new("color", "red"));
 ```
 
+### Changing the cardinality limit for a MeterProvider
+
+To set the default [cardinality limit](../README.md#cardinality-limits) for all
+metrics managed by a given `MeterProvider`, use the
+`MeterProviderBuilder.SetMaxMetricPointsPerMetricStream` extension:
+
+> [!CAUTION]
+> `MeterProviderBuilder.SetMaxMetricPointsPerMetricStream` is marked `Obsolete`
+  in pre-release builds and has been replaced by
+  `MetricStreamConfiguration.CardinalityLimit`. For details see:
+  [OTEL1003](../../diagnostics/experimental-apis/OTEL1003.md).
+
+```csharp
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter("*")
```

Comment:
Could we avoid `AddMeter("*")` in all the docs/examples? As more libraries are adding instrumentations, I guess we don't expect anyone to enable _all_.

Reply:
This is what I would suggest: https://github.com/open-telemetry/opentelemetry-dotnet/blob/39d960e0e77aa81c022bda0b33e1d23cf93a9f2d/docs/metrics/customizing-the-sdk/README.md?plain=1#L383

Reply:
I fixed it here and the one above